spec/controllers/api/v0x0/tasks_controller_spec.rb

```diff
@@ -0,0 +1,19 @@
+RSpec.describe Api::V0x0::TasksController, :type => :request do
```

Comment:
Can you add the same tests for v0.1?  This feels like it should be an independent PR.

Reply:
Moved to #137